src/main/java/com/xjeffrose/xio/server/XioServerConfig.java

```diff
@@ -5,112 +5,40 @@
 import com.xjeffrose.xio.SSL.TlsConfig;
 import io.netty.channel.ChannelOption;
 import io.netty.util.Timer;
+import lombok.Getter;
 
 import java.util.Map;
 import java.util.concurrent.ExecutorService;
 import java.net.InetSocketAddress;
 
 public class XioServerConfig {
-  // old
+  @Getter
```

Comment:
Could a class level @Getter annotation be used here?  Is that being avoided so that any future fields don't accidentally get exposed with a getter?